qcinbm/awesome-ai-models
# Feature/activation functions

This PR introduces the following changes:
- Added a new file `activation_functions.py` under `models/utils` to encapsulate common activation functions used in machine learning.
- Includes implementations for:
  - Sigmoid
  - ReLU
  - Leaky ReLU
  - Tanh
  - Softmax
- Added a `test.ipynb` notebook to verify and demonstrate the functionality of these activation functions.
- Updated `environment.yml` to include necessary dependencies for the project.

### How to Test
1. Activate the project environment: `conda activate awesome-env`.
2. Run `test.ipynb` to verify outputs of activation functions.
3. Integrate the functions into existing deep learning models.

This feature is a key step for building reusable and modular machine learning components.
